packages/sanity/src/core/studio/components/navbar/resources/ResourcesMenuItems.tsx

```diff
@@ -0,0 +1,123 @@
+import {Box, Card, Flex, Label, MenuDivider, MenuItem, Spinner, Text} from '@sanity/ui'
+import React from 'react'
+import {SANITY_VERSION} from '../../../../version'
+import {ResourcesResponse, Section} from './helper-functions/types'
+
+interface ResourcesMenuItemProps {
+  error: Error | null
+  isLoading: boolean
+  value: ResourcesResponse | undefined
+}
+
+export function ResourcesMenuItems({error, isLoading, value}: ResourcesMenuItemProps) {
+  const sections = value?.resources?.sectionArray
+  const latestStudioVersion = value?.latestVersion
+
+  return (
+    <>
+      {isLoading ? (
+        <Flex align="center" justify="center" padding={3}>
+          <Spinner />
+        </Flex>
+      ) : (
+        <>
+          {/* Display fallback values on error / no response */}
+          {(value === undefined || error) && <FallbackLinks />}
+
+          {!error &&
+            sections?.map((subSection) => {
+              if (!subSection) return null
+              return <SubSection key={subSection._key} subSection={subSection} />
+            })}
+
+          {/* Studio version information */}
+          <Box padding={3}>
+            <Text size={1} muted weight="medium" textOverflow="ellipsis">
+              Sanity Studio version {SANITY_VERSION}
+            </Text>
+            {!error && !isLoading && latestStudioVersion && (
+              <Box paddingTop={2}>
+                <Text size={1} muted textOverflow="ellipsis">
+                  Latest version is {latestStudioVersion}
+                </Text>
+              </Box>
+            )}
+          </Box>
+        </>
+      )}
+    </>
+  )
+}
+
+const FallbackLinks = () => (
```

Comment:
This can/should be made into a constant instead of a function - saves React from creating and diffing it on each render.

Psuedo-code:
```diff
-const FallbackLinks = () => <>...menuItems...</>
+const fallbackLinks = <>...menuItems...</>

- <div><FallbackLinks /></div>
+ <div>{fallbackLinks}</div>
```